impl-tinkerpop-parent/impl/src/main/java/org/hawkular/inventory/impl/tinkerpop/TenantBrowser.java

```diff
@@ -72,7 +74,7 @@ private TenantBrowser(InventoryContext context, FilterApplicator.Tree path) {
             }
 
             @Override
-            public Tenant entity() {
+            public Tenant entity() throws EntityNotFoundException, RelationNotFoundException {
```

Comment:
So `{Entity|Relation}NotFoundException` are now checked, or is it just about being more explicit?